src/openfga.ne

```diff
@@ -1,11 +1,25 @@
 @preprocessor typescript
 
 types           -> (_newline):* (type (relations (define):+):*):* {%
-    // @ts-ignore
-    data => data[1].map((datum) => {
+    data => {
+         // @ts-ignore
+		const types = data[1].map((datum : any) => {
 		    const relations = datum[1][0] ? datum[1][0][1].flat() : [];
 		    return { ...datum[0], relations }
 		})
+		let schemaVersion = "1.0"
+        // @ts-ignore
+		types.forEach(type => {
```

Comment:
```javascript
const hasTypes = types.some(type => type.relations.some(relation => relation.allowedTypes.length));
const schemaVersion = hasTypes ? "1.1" : "1.0";
```

Just so it is clearer and so that you don't have to loop through all the types/relations - some allows you to exit as soon as you find a match